security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -107,7 +107,7 @@ public Options getOptions() {
                         .argName("FILE")
                         .build());
                 options.addOption(Option.builder().longOpt(WITH_EXTENSIONS_OPTION)
-                        .desc("the extension list to enable")
+                        .desc("the extension list to consistent")
```

Comment:
consistent is not a verb?

Reply:
That was an error when i used the intellij rename all option.